src/jwt.cpp

```diff
@@ -109,10 +115,15 @@ Validator stringValidator(std::string&& name,
 
 std::string formatTime(std::time_t value) noexcept
 {
-    std::tm tmb;
     char buf[20];
-    gmtime_r(&value, &tmb);
-    auto res = std::strftime(buf, sizeof(buf), "%F %T", &tmb);
+#ifdef WIN32
+	std::tm * tmb=nullptr;
+    tmb = gmtime(&value);
+#else 
+	std::tm tmb;
+    gmtime_r(&value, tmb);
+#endif
+    auto res = std::strftime(buf, sizeof(buf), "%F %T", tmb);
```

Comment:
Depending on compiler options, 'tmb' here is either a value or a pointer. std::strftime takes it by pointer.